transactions/set-version-boundary/2024/oct-8/arguments.json

```diff
@@ -12,11 +12,8 @@
     "value": "17"
   },
   {
-    "type": "Optional",
```

Comment:
we ran the HCU by updating the transaction and making the release version as non-optional.